### DISCUSSION TITLE
Feature request: Snow timer

### DISCUSSION THREAD

**Discussion by Yiaro222** [runelite/runelite#15055] [Ideas]
A timer when using a snowglobe or reindeer hat, a timer for when to put the snow down and a timer for when the snow will be gone. and/or need to place another snow layer again.

- Used for f2p tick manipulation
- Used for snow collections

**Comment by Nightfirecat** [comment]
@Yiaro222 You can already use the shift-click configuration plugin to access that option more easily.

**Comment by Nightfirecat** [comment]
Ideally, when #1556 is merged, that can be resolved more nicely.

**Comment by geheur** [comment]
There is a hub plugin for this: https://github.com/buttwize/runelite-snow-tracker-plugin
